src/libstd/ascii.rs

```diff
@@ -246,7 +324,7 @@ pub struct EscapeDefault {
     data: [u8; 4],
 }
 
-/// Returns a 'default' ASCII and C++11-like literal escape of a `u8`
+/// Returns an iterator that produces an escaped version of a `u8`.
```

Comment:
'default' is mentioned later too, could it be given some better context/antecedent?